poetry2nix: 1.13.0 -> 1.14.0

adisbladis opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

adisbladis committed:
poetry2nix: 1.13.0 -> 1.14.0

SuperSandro2000 commented on pkgs/development/tools/poetry2nix/poetry2nix/pkgs/poetry/src.json:
```diff
@@ -1,7 +1,7 @@
+    "sha256": "0lx3qpz5dad0is7ki5a4vxphvc8cm8fnv4bmrx226a6nvvaj6ahs",
     "fetchSubmodules": true
-}
+}
```
Missing new line.

SuperSandro2000 commented:
Result of `nixpkgs-review pr 105477` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>pinnwand</li>
    <li>poetry</li>
  </ul>
</details>

SuperSandro2000 commented:
Result of `nixpkgs-review pr 105477` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>pinnwand</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>poetry</li>
  </ul>
</details>

pinndwan: basically all tests fail on darwin:

```
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_many_too_large
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_mismatched
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_multiple
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_nonexistent_expiry
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_nonexistent_lexer
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_nothing
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_only_filenames
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_only_lexers
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_only_raws
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_only_xsrf
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_raw_only_space
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_single
FAILED test/test_http_website.py::WebsiteTestCase::test_website_create_post_too_large
FAILED test/test_http_website.py::WebsiteTestCase::test_website_expiry - Perm...
FAILED test/test_http_website.py::WebsiteTestCase::test_website_hex_nonexistent_paste
FAILED test/test_http_website.py::WebsiteTestCase::test_website_index - Permi...
FAILED test/test_http_website.py::WebsiteTestCase::test_website_index_with_lexer
FAILED test/test_http_website.py::WebsiteTestCase::test_website_index_with_nonexistent_lexer
FAILED test/test_http_website.py::WebsiteTestCase::test_website_nonexistent_page
FAILED test/test_http_website.py::WebsiteTestCase::test_website_raw_nonexistent_paste
FAILED test/test_http_website.py::WebsiteTestCase::test_website_removal - Per...
FAILED test/test_http_website.py::WebsiteTestCase::test_website_show_nonexistent_paste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_download
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_download_nonexistent_paste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_hex
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_hex_nonexistent_paste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_empty_code
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_no_code
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_no_expiry
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_no_lexer
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_nonexistent_expiry
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_index_post_nonexistent_lexer
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_logo
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_raw
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_remove
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_remove_nonexistent_paste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_repaste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_repaste_nonexistent_paste
FAILED test/test_http_website.py::DeprecatedWebsiteTestCase::test_website_show
FAILED test/test_utility.py::test_guess_language - OSError: [Errno 24] Too ma...
======================== 88 failed, 11 passed in 8.23s =========================
```

adisbladis committed:
pinnwand: Use poetry from pythonPackages

Ideally this software should be repackaged using Poetry2nix rather
that using pythonPackages.

As it is now things are packaged incorrectly.

cc @mweinelt

mweinelt commented:
Thanks for fixing that up. I'll look into poetry2nix soon.
